XamarinCommunityToolkit/Behaviors/Validators/CharacterType.shared.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	[Flags]
+	public enum CharacterType
+	{
+		LowerLetter = 1,
+		UpperLetter = 2,
```

Comment:
I feel that it would improve readability if you changed these enums to `LowercaseLetter` and `UppercaseLetter`.

Reply:
Nice point, I'll amend